test/index.js

```diff
@@ -252,6 +269,20 @@ describe('rechoir', function() {
 
       done();
     });
-  });
 
+    it('should register a module loader for the specified extension with cwd', function(done) {
```

Comment:
New test?

Reply:
Yes. there was no case about cwd option and this case make coverage to 100%.